tests/Pagination.spec.js

```diff
@@ -23,9 +23,10 @@ describe('Uncontrolled Pagination', () => {
   function shouldHighlightRight() {
     const pagers = TestUtils.scryRenderedDOMComponentsWithTag(pagination, 'li');
     const current2 = pagination.state.current;
+    const totalText = 1;
```

Comment:
I think some thing wrong on the test case.
Why master branch don't pass the 'npm run chrome-test'.
I try to complete my pull request and found the test case is not pass in the master.
Do you want me to fix the master code make it pass test case don't change the test case right?
Can you suggest me @benjycui am a beginner on this.